doc/nvim-tree-lua.txt

```diff
@@ -194,9 +194,10 @@ Requirements
 
 *:NvimTreeGenerateOnAttach*
 
-    Creates and opens a new file `/tmp/my_on_attach.lua` containing an
-    |nvim-tree.on_attach| function based on your |nvim-tree.view.mappings|,
-    |nvim-tree.remove_keymaps| as well as the defaults.
+    Creates and opens a new file `my_on_attach.lua` in |stdpath| `"cache"`,
```

Comment:
Should we put it in `run` instead?

Per dos:
```
		run          String  Run directory: temporary, local storage
				     for sockets, named pipes, etc.
``

Reply:
`$XDG_RUNTIME_DIR` `/run/user/1000` 

Looks like it should be sane on windows however I'd rather not take the risk...